src/main/java/CellContent.java

```diff
@@ -0,0 +1,97 @@
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellType;
+
+public class CellContent {
+	public CellType type;
```

Comment:
Don't worry - I am not a OOP nazi. This class indeed makes sense, for example to implement the behaviour suggested below `cell.setCellValue(content.getValue());`. The _I personally think trusting the programmers_ argument is sometimes critical, especially if there is not too much documentation present. I am not a C++ professional, but I guess C++ folks also use accessors in case there is encapsulation or additional state handling required (before accessing the corresponding member). In general, it makes sense to ease the code complexity as much as possible but the same time prevent a programmer from introducing incorrect states.